sbin/pfdetect

```diff
@@ -156,10 +156,11 @@ sub _run_detector {
         my $line = $_;
         $logger->info("alert received: '$line'");
         my $data = $parser->parse($line);
-        unless($data) {
+        unless($data || $data eq 0) {
```

Comment:
He will definitely not see email notification(if enabled), since he doesn't read email either!